tests/interchain/matrix_tool/main.go

```diff
@@ -23,6 +23,13 @@ import (
 
 const branchBuildSuffix = "-ref"
 
+func GetUpgradeName(testVersion string) string {
```

Comment:
```suggestion
func getUpgradeName(testVersion string) string {
```

nit - doesnt need to be exported